src/main/java/me/ryanhamshire/GriefPrevention/Claim.java

```diff
@@ -731,17 +731,20 @@ public boolean contains(Location location, boolean ignoreHeight, boolean exclude
         //not in the same world implies false
         if (!Objects.equals(location.getWorld(), this.lesserBoundaryCorner.getWorld())) return false;
 
+        BoundingBox boundingBox = new BoundingBox(this);
```

Comment:
not sure if this will cause extra GC pressure (unsure where this is called - is it called in all cases of blockbreakevent and etc.?)

Reply:
This was previously always created on L744.

Reply:
oh ok

Reply:
You're right though, that is food for thought - I really do hope to eventually just make Claim extend BoundingBox, honestly.